src/components/VContentSwitcher/VContentTypeButton.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <button
+    type="button"
+    class="transition-colors"
+    :class="[$style.button, selected && $style['button-pressed']]"
+    :aria-pressed="selected"
+    @click="handleClick"
+  >
+    <VIcon :icon-path="iconPath" class="me-1 flex-shrink-0" />
+    <span>{{ $t(`search-type.${contentType}`) }}</span>
+  </button>
+</template>
+
+<script>
+import { computed, defineComponent } from '@nuxtjs/composition-api'
+import {
+  ALL_MEDIA,
+  AUDIO,
+  IMAGE,
+  supportedContentTypes,
+} from '@/constants/media'
+import VIcon from '@/components/VIcon/VIcon.vue'
```

Comment:
I don't know what the difference is or why to prefer one over the other, but we use `~` almost everywhere else, so it might be good to stick with that to prevent any confusion for new contributors.

```suggestion
} from '~/constants/media'
import VIcon from '~/components/VIcon/VIcon.vue'
```

Reply:
> 

It's Webstorm! Whenever I use automatic refactoring or rely on it to add imports, it uses @. I try to change it every time, but here I missed it. There should probably be a setting for it...